bin/travis/before_script_php.sh

```diff
@@ -14,5 +14,9 @@ fi
 echo "memory_limit=2048M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
 phpenv config-rm xdebug.ini
 composer self-update
+echo "Install prestissimo"
+composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --optimize-autoloader --classmap-authoritative  --no-interaction
+echo "Install dependencies ..."
```

Comment:
I wouldn't add `echo`s here... The log is already cluttered enough, and the command tells pretty good what's happening